tools/custom-organization-policy-library/build/custom-constraints/gke/gkeAllowedReleaseChannels.yaml

```diff
@@ -1,12 +1,15 @@
 #@ load("/constraints.lib.star", "build_constraint")
 #@ constraint = build_constraint("gkeAllowedReleaseChannels")
 
+#@ def condition(release_channels):
+#@   return 'resource.releaseChannel.channel != "' + release_channels + '"'
```

Comment:
U need to have that kind of logic:
checking that resource.releaseChannel.channel is included in release_channels
```
resource.releaseChannel.channel in ["STABLE", "REGULAR"]
```
```
return 'resource.releaseChannel.channel in' +  str(release_channels)
```